src/components/UsernameEntry/UsernameEntry.js

```diff
@@ -21,7 +22,12 @@ class UsernameEntry extends React.Component {
 				<div className="text-select-slack">
 					Please select your slack handle to send a reminder
 				</div>
-				<div>~list component~</div>
+				<div>
+					<ButtonList
+						items={this.props.users}
+						onClick={(id, name) => this.sendReminder(name)}
```

Comment:
It's preferable not to call the props onClick because onClick is an actual thing for a button. An onClick would usually call a function called handleClick or something, but its not really to do with this PR so it's fine.